internal/setup/setup.go

```diff
@@ -157,12 +176,35 @@ func (r *Release) validate() error {
 						}
 						return fmt.Errorf("slices %s and %s conflict on %s", old, new, newPath)
 					}
-				} else {
-					if newInfo.Kind == GlobPath {
-						globs[newPath] = new
+					// Note: We do not have to record newPath because conflict
+					// is a transitive relation.
```

Comment:
Discussed offline, the code and its assumptions were okay, the same ones we had on master basically. The problem was the comment which attempted to explain a nuanced relation in a very sort sentence. I have changed the comment to something that captures the intent with more precision because it is true that the conflict is NOT a transitive relation, it is more akin to "equivalence classes" of no-conflict where we partition by paths. However, that is again too complex so I have written the comment in the most straight-forward way I could think of.